src/lua/httpc.lua

```diff
@@ -529,6 +529,8 @@ end
 --
 --      params - a table with query parameters;
 --
+--      http_version - describes a http version;
```

Comment:
Nit: `accept_encoding` looks as the latest added request option except one that enables a specific behavior like `chunked`.

I would place this comment right after the `accept_encoding` so.

Also, I would place the `lua_getfield(L, 5, "http_version");` block after the `lua_getfield(L, 5, "accept_encoding");` block, the `httpc_set_http_version` function declaration after the `httpc_set_accept_encoding` function declaration.

Such consistency helps with a code readability.

Reply:
Fixed